jupyter_book/tests/test_pdf.py

```diff
@@ -15,3 +15,11 @@ def test_html_pdf(tmpdir):
     path_pdf = path_output.joinpath("_build", "pdf")
     assert path_html.joinpath("index.html").exists()
     assert path_pdf.joinpath("book.pdf").exists()
+
+def test_pdflatex(tmpdir):
+    path_output = Path(tmpdir).absolute()
+    path_template = path_tests.parent.joinpath("book_template")
+    cmd = f"jb build {path_template} --path-output {path_output} --build pdflatex"
+    run(cmd.split(), check=True)
+    path_pdf = path_output.joinpath("_build", "latex")
+    assert path_pdf.joinpath("python.pdf").exists()    #TODO: Update name of file once configured for named latex document
```

Comment:
the name of the file that `pdfhtml` outputs is always called `book.pdf`, can we just start w/ that convention as a default?

Reply:
the default `latexpdf` writer in sphinx doesn't use the usual names when building the tex document. Figuring out how to pass `book.pdf` through in the config.